Bearded.Utilities/IO/Logger.cs

```diff
@@ -30,20 +30,20 @@ internal Writer(Logger logger, Severity severity)
             }
 
             public void Log<T>(T obj) {
-                logger.AddEntry(new Entry(obj.ToString(), severity));
+                logger.AddEntry(new Entry(obj?.ToString() ?? "null", severity));
             }
 
             public void Log(string text, params object[] parameters) {
                 Log(string.Format(text, parameters));
             }
         }
 
-        public Writer Fatal { get; }
-        public Writer Error { get; }
-        public Writer Warning { get; }
-        public Writer Info { get; }
-        public Writer Debug { get; }
-        public Writer Trace { get; }
+        public Writer? Fatal { get; }
+        public Writer? Error { get; }
+        public Writer? Warning { get; }
+        public Writer? Info { get; }
+        public Writer? Debug { get; }
+        public Writer? Trace { get; }
```

Comment:
This is the only real breaking change of this PR. We always intended the logger to be used like `logger.Warning?.Log()` so this is finally enforcing that to be best of our ability.